### DISCUSSION TITLE
Welcome to Unified Graphics Discussions!

### DISCUSSION THREAD

**Discussion by ian-noaa** [NOAA-GSL/unified-graphics#6] [Announcements]
## 👋 Welcome!
  We’re using GitHub's Discussions as the central location for communication related to the Unified Graphics application. We're hoping that the majority of project-related communications can happen here so that, if needed, we can tag other people in for their input. 

We've created the following categories:
  * **Announcements**: Category for important communications from the project to others in the community.
  * **Development**. Category for regular conversations within the team about the development of the application.
  * **Ideas**. Category for suggestions and requests from stakeholders.
  * **Help**. Category for users to ask questions and get help with the software.

**Comment by esheehan-gsl** [comment]
If you want to follow along with the development of this project, you may want to subscribe to notifications from this repository using the “Watch” menu in the top right.

![The four watching options available in the “Watch” menu: participating and metions, all activity, ignore, and custom](https://user-images.githubusercontent.com/85259329/164020569-bd5e0192-b693-4f98-b936-23ae8e870ff3.png)

If you don’t want to be notified about every little thing that happens in the repository, you can choose “Custom” and just turn on “Discussions” so that you’ll be notified about new discussion topics. You can learn more about GitHub’s notifications from the [GitHub documentation](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).
